Code/Mantid/Framework/DataObjects/src/ReflectometryTransform.cpp

```diff
@@ -223,8 +229,27 @@ Mantid::API::MatrixWorkspace_sptr ReflectometryTransform::execute(
   return ws;
 }
 
+boost::optional<boost::shared_ptr<Mantid::DataObjects::TableWorkspace>>
```

Comment:
It would be best if the Type for this argument was not boost::optional<boost::shared_ptr<Mantid::DataObjects::TableWorkspace>> but was instead Mantid::DataObjects::Table&